XML3-Homework/parrish_1-26_xml3.xml

```diff
@@ -0,0 +1,115 @@
+            needle. </stitchSteps>
+        <stitchSteps step="third"> 3 How to knit step 3 Slide the right needle back bringing the
+            yarn with it, so the right needle sits on top of the left. </stitchSteps>
+        <stitchSteps step="fourth"> 4 How to knit step 4 Slide the stitch off the left hand needle
+            so you now have one knitted stitch on your right. You’ve completed your first knit
+            stitch. Go you! </stitchSteps>
+        <stitchSteps step="fifth"> 5 Knit your first row Repeat your knit stitch until you’ve
+            completed the first row! </stitchSteps>
+        <stitchSteps step="sixth"> 6 Turn your right needle around and transfer it to your left
+            hand. Swap your empty left hand needle into your right hand to begin stitching your
+            second row using steps 1-5. </stitchSteps>
+        <stitchSteps step="seventh"> 7 Repeat the number of rows you’d like to build your fabric.
+        </stitchSteps>
+    </knitStitchSteps>
+
+    <mainSteps>
+        <step main="sixth">6. Bind off</step> Binding off is the final knitting technique you’ll
+        need to complete your first knitting project. While there are lots of ways to bind off,
+        we’re going to show you a simple bind off method to get you started. </mainSteps>
+
+    <bindOffSteps> To ‘bind off’ just means fastening up your live stitches using your needles, so
+        you have a nice neat and secure edge. Once you’ve got this finishing technique in the bag
+        you're ready to start your first project! <header>How to make a simple bind off</header>
+        <bindOff step="first"> 1 How to cast off step 1 Knit your first stitch, as if you were
+            beginning a new row. </bindOff>
+        <bindOff step="second"> 2 How to cast off step 2 Knit the second stitch so you have two
+            stitches on your right needle. </bindOff>
+        <bindOff step="third"> 3 How to cast off step 3 Use the tip of your left needle to insert
+            underneath and into the first stitch on your right needle. </bindOff>
+        <bindOff step="fourth"> 4 How to cast off step 4 Use the left needle to lift the first
+            stitch and pull it over the second, and that’s it! You’ve completed your first bind off
+            stitch. </bindOff>
+        <bindOff step="fifth"> 5 How to cast off step 5 Repeat this bind off method all the way to
+            the end of your row. Once you’ve reached the end use your scissors to snip the working
+            yarn leaving a tail. Pull the tail through the final loop and tighten to secure a knot. </bindOff>
+        <bindOff step="sixth"> 6 Now you’ve finished binding off, sew in your ends to complete!
+        </bindOff>
+    </bindOffSteps>
+
+    <extraInfo> Knitting videos for beginners Get started with the knitting basics! If you've never
+        picked up a pair of needles before, the thought of learning to knit can seem pretty
```

Comment:
@kzp308 Here is, the "Knitting videos for beginners" basically a section heading? Maybe that could go in an element:

```xml
<extraInfo>
     <heading>Knitting videos for beginners</heading>
     Get started with ...
 </extraInfo>
 ```